### DISCUSSION TITLE
Node file names?

### DISCUSSION THREAD

**Discussion by macdroid53** [j8sr0230/fc_nodes#105] [Q&A]
I did a couple node graphs and saved.

They have no file extension.

And, won't open from the Open menu.

I tried renaming with the extension .Graph but that didn't work...

**Comment by j8sr0230** [comment]
Thanks for your report! I just replied to you in the FreeCAD forum.  Actually, the files should be automatically saved as JSON files. Can you open them when you append *.json to your files?

**Reply by macdroid53** [reply]
Detailed reply: https://forum.freecadweb.org/viewtopic.php?p=644969#p644969
